tensorflow/python/ops/array_ops.py

```diff
@@ -4482,7 +4482,7 @@ def where_v2(condition, x=None, y=None, name=None):
 
   Args:
     condition: A `tf.Tensor` of type `bool`
-    x: If provided, a Tensor which is of the same type as `y`, and has a shape
+    x: If provided, a Tensor which is of the same type as `x`, and has a shape
       broadcastable with `condition` and `y`.
     y: If provided, a Tensor which is of the same type as `y`, and has a shape
       broadcastable with `condition` and `x`.
```

Comment:
The documentation wants to say that `x` and `y` have the same type. So `x` has the same type as `y` and then `y` has the same type as `x`.

Reply:
**Updated the changes as requested in pull request #41044**